package.json

```diff
@@ -44,6 +44,7 @@
     "@babel/preset-env": "^7.16.0",
     "@babel/preset-react": "^7.16.0",
     "@babel/register": "^7.16.0",
+    "@floating-ui/react-dom": "^0.7.1",
```

Comment:
I think this should be in the `dependencies` section?
Maybe `peerDependencies` if it is not mandatory.

Reply:
I think it's correct.

- It's only used in the docs as an example implementation—that's why `devDependencies`.
- It's just a **recommended** library to use along with `Popover`. You can use RUI with plain `Popover` (with no library) or even without `Popover`.

There is an [`optionalDependencies` field](https://docs.npmjs.com/cli/v8/configuring-npm/package-json#optionaldependencies) in npm, but it does a different thing: installation just will not fail if such dependencies cannot be found or installed.

In my opinion, it's an **opt-in dependency**, therefore I think it's correct just to list it in the docs.

Reply:
OK, I get it now.
Somehow I thought the integration with `floating-ui` is tighter. I guess the real issue is the docs then and that is addressed in another comment.

Thank you